src/py42/sdk/queries/fileevents/file_event_query.py

```diff
@@ -34,14 +34,20 @@ def __str__(self):
         groups_string = u",".join(
             str(group_item) for group_item in self._filter_group_list
         )
-        json = u'{{"groupClause":"{0}", "groups":[{1}], "pgNum":{2}, "pgSize":{3}, "srtDir":"{4}", "srtKey":"{5}"}}'.format(
-            self._group_clause,
-            groups_string,
-            self.page_number,
-            self.page_size,
-            self.sort_direction,
-            self.sort_key,
-        )
+
+        items = []
```

Comment:
This can be simplified:
```python
if self.page_token is not None:
    paging_prop = '"pgNum":{0}'.format(self.page_number)
else:
    paging_prop = '"pgToken":{0}'.format(self.page_token)

json = u'{{"groupClause":"{0}", "groups":[{1}], "srtDir":"{2}", "srtKey":"{3}", {4} }}'.format(
    self._group_clause,
    groups_string,
    self.sort_direction,
    self.sort_key,
    self.paging_props
)


```

Reply:
Changed.